fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1210,6 +1210,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static boolean enable_replicated_storage_as_default_engine = true;
 
+    /**
+     * enable schedule insert query by row count
+     */
+    @ConfField(mutable = true)
+    public static boolean enable_schedule_insert_query_by_row_count = true;
```

Comment:
It seems to use the row count instead of quantity is reasonable. Should we deprecate the old quantity-based schedule algorithm?

Reply:
Wait for this insert into scene to run stable.